packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -124,9 +124,9 @@ export class HostedPluginProcess implements ServerPluginRunner {
 
     killProcessTree(parentPid: number): void {
         psTree(parentPid, (_, childProcesses) => {
-            childProcesses.forEach(childProcess =>
-                this.killProcess(parseInt(childProcess.PID))
-            );
+            childProcesses.forEach(childProcess => {
+                this.killProcess(parseInt(childProcess.PID));
+            });
```

Comment:
The change is unnecessary, it can be a single line.

Reply:
Fixed